### PR TITLE
[NBug] An error occurred trying to start process

### PR DESCRIPTION
**Pull request by SaumyaBhushan**
[workerB](https://workerb.linearb.io/v2/badge/collaboration-page?magicLinkId=2x-D5oH)

Fixes https://github.com/gitextensions/gitextensions/issues/10429

## Proposed changes

- url is now a 404. Now point to https://github.com/acaudwell/Gource#readme now.

### Before

[NBug] An error occurred trying to start process

### After
Resolved the issue by replacing old link

## Test environment(s) 

- GIT 
- Windows 

## Merge strategy

I agree that the maintainer squash merge this PR (if the commit message is clear).

----

:black_nib: I contribute this code under [The Developer Certificate of Origin](../blob/master/contributors.txt).

### CODE REVIEW

File: Plugins/Gource/GourceStart.cs [hunk 1]
```diff
@@ -153,7 +153,7 @@ private void linkLabel1_LinkClicked(object sender, LinkLabelLinkClickedEventArgs
 
         private void linkLabel2_LinkClicked(object sender, LinkLabelLinkClickedEventArgs e)
         {
-            Process.Start(@"https://github.com/acaudwell/Gource/blob/master/README");
+            Process.Start(@"https://github.com/acaudwell/Gource#readme");
```
Comment (vbjay) [inline Plugins/Gource/GourceStart.cs:156]: I recommend using https://github.com/gitextensions/gitextensions/blob/master/GitUI/OsShellUtil.cs#L45.

**Review by vbjay — CHANGES_REQUESTED** [review]

**Comment by SaumyaBhushan** [comment]
Done @RussKie

File: Plugins/Gource/GourceStart.cs (hunk 1 above)
Comment (SaumyaBhushan) [reply Plugins/Gource/GourceStart.cs:156]: Hi @vbjay I am not sure how can I use this ? Can you please guide me ?

Comment (pmiossec) [reply Plugins/Gource/GourceStart.cs:156]: In cases like that, a good habit is to have a look at how it is already used and you will find something like that:
`OsShellUtil.OpenUrlInDefaultBrowser(@"https://github.com/acaudwell/Gource#readme");`

Comment (SaumyaBhushan) [reply Plugins/Gource/GourceStart.cs:156]: Thanks @pmiossec.

**Review by pmiossec — APPROVED** [review]

**Review by mstv — APPROVED** [review]
Thank you

**Pull request merged**